### DISCUSSION TITLE
Is there any Debian analogue of configuration scripts available?

### DISCUSSION THREAD

**Discussion by smabbasht** [prasanthrangan/hyprdots#519] [General]
Hey, 
I was wondering if there's any analogue of the setup scripts for Debian based distributions like Ubuntu etc. If there is none, can this be done and will any contribution be appreciated towards that?
Actually, I am an Archlinux user and my friends were already fascinated by how fast I am able to use my laptop when I used to have i3wm.  now that I have switched to hyprland, seeing your config, they keep asking me to set their systems like this, fascinated by the immense speed and bleeding aesthetics, btw, they all use Ubuntu.
Any pointers would be much appreciated

**Comment by Balssh** [comment]
Probably you missed it, but it was the first line in the readme: https://github.com/prasanthrangan/hyprdots#installation
He links this repo for debian: https://github.com/Senshi111/debian-hyprland-hyprdots
Beware thought, I'm not sure how up to date that repo is.

**Comment by kRHYME7** [comment]
Hello @smabbasht :wave:,

We have a discussion tab in this repo where you can ask questions or suggest features. As we have a single maintainer for hyprdots, my concerns are I want to help keep the repo clean and manageable.

Also, I will use this message template for future replies like this. Haha :grinning:  Thank you for understanding :pray: 

The instructions above should guide you. 
Cheers!

``` 
Hey,

We appreciate and are happy to have your input. However, our issue tracker is mainly for problems related to the configs. If you're looking to suggest something unrelated to the configs, need guides, or just want to chat, we have a discussion tab which is the perfect place for that.

We're just trying to keep things tidy here and a single maintainer is handling most of the stuff.

Thanks for understanding!

[khing] (Contributors with more errors HAha)
```

**Comment by prasanthrangan** [comment]
Hey @smabbasht , thank you so much!
Please refer @Senshi111's version, he did make it work with debian sid...
So I'll move this to discussions!!

**Reply by Senshi111** [reply]
So what i did i separated hyprdots files to another repo so i can use it in multi distros and it's easier for me to update hyprdots file in one place.
I'm on fedora now so my debian and fedora install scripts pulling my hyprdots files and installing them. I make them undependent from distro. Sure i had to for now removed some minor things from some scripts related to arch like systemupdate.sh etc

**Comment by smabbasht** [comment]
Thankyou all and sorry for not looking up the readme so keenly. I don't know how I missed that. I will look into the provided repository.

**Comment by Senshi111** [comment]
Last time i tried on Ubuntu it didn't work on first run. I had to add some extra dependencies for hyprland build because they are not completely the same as for debian. But after i add those dependencies, everything build just fine
